sources/packages/backend/apps/test-db-seeding/src/seed-executors/seed-executor.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  DiscoveredClassWithMeta,
+  DiscoveryService,
+} from "@golevelup/nestjs-discovery";
+import { Injectable, OnModuleInit } from "@nestjs/common";
+import { DATA_SEED, DATA_SEED_METHOD } from "./data-seed.decorator";
+
+@Injectable()
+export class SeedExecutor implements OnModuleInit {
+  constructor(private readonly discoveryService: DiscoveryService) {}
+
+  async onModuleInit(testServicesClasses?: string[]): Promise<void> {
+    // Discovers all providers in the test-db-seeding app that have meta at a specific
+    // key and returns the provider(s) and associated meta.
+    const discoveredClasses =
+      await this.discoveryService.providersWithMetaAtKey<{
```

Comment:
Can we please have an interface created for this?
`{
        name: string;
        order: number;
        skip: boolean;
}`
For these decorators, I would suggest using the options approach with the suggested interface.
Then the decorator can be declared as, for instance, `@DataSeed({skip; true})`.